scripts/ecs/appsignals/setup-ecs-demo.sh

```diff
@@ -0,0 +1,1124 @@
+VPC_ID="vpc-0000000000aaaaaaa"
+SUBNET_IDS="subnet-0000000000aaaaaaa subnet-1111111111bbbbbbb"
+SECURITY_GROUP_ID="sg-0000000000aaaaaaa"
+LOAD_BALANCER_ARN="arn:aws:elasticloadbalancing:${REGION}:000111222333:loadbalancer/app/${LOAD_BALANCER_NAME}"
+LOAD_BALANCER_DNS="ecs-pet-clinic-lb-${REGION}-000111222333.us-east-1.elb.amazonaws.com"
+ECR_IMAGE_PREFIX="000111222333.dkr.ecr.us-east-1.amazonaws.com"
+
+function create_resources() {
+    echo "Creating resources..."
+    # Get the default VPC
+    VPC_ID=$(aws ec2 describe-vpcs --filters "Name=isDefault,Values=true" --query 'Vpcs[0].VpcId' --output text)
+
+    SUBNET_IDS=$(aws ec2 describe-subnets --filters "Name=vpc-id,Values=$VPC_ID" --query "Subnets[*].SubnetId" --output text)
+    echo "Subnet IDs: $SUBNET_IDS"
+
+    # Create a security group
+    SECURITY_GROUP_ID=$(aws ec2 create-security-group --group-name $SG_NAME --description "Security group for all traffic" --vpc-id $VPC_ID --query 'GroupId' --output text)
+    aws ec2 authorize-security-group-ingress --group-id $SECURITY_GROUP_ID --protocol all --cidr 0.0.0.0/0
+    echo "Security group ID: $SECURITY_GROUP_ID"
+
+    # Create a load balancer
+    LOAD_BALANCER_ARN=$(aws elbv2 create-load-balancer \
+        --name $LOAD_BALANCER_NAME \
+        --subnets $SUBNET_IDS \
+        --security-groups $SECURITY_GROUP_ID \
+        --scheme internet-facing \
+        --type application \
+        --query "LoadBalancers[0].LoadBalancerArn" \
+        --output text)
+    echo "Load balancer ARN: $LOAD_BALANCER_ARN"
+
+    LOAD_BALANCER_DNS=$(aws elbv2 describe-load-balancers \
+        --load-balancer-arns $LOAD_BALANCER_ARN \
+        --query "LoadBalancers[0].DNSName" \
+        --output text)
+
+    echo "Load balancer DNS: $LOAD_BALANCER_DNS"
+
+
+    # Create an ECS Task role and attach policies
+    aws iam create-role --role-name $IAM_TASK_ROLE_NAME --assume-role-policy-document file://trust-policy.json > /dev/null
+    aws iam attach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn "arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess"
+    aws iam attach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn "arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess"
+    aws iam attach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
+    aws iam attach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole"
+    aws iam attach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn "arn:aws:iam::aws:policy/AmazonECS_FullAccess"
+    aws iam attach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn "arn:aws:iam::aws:policy/AmazonSQSFullAccess"
+    aws iam attach-role-policy --role-name $IAM_TASK_ROLE_NAME --policy-arn "arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess"
+
+    # Create an ECS Task Execution role and attach policies
```

Comment:
Why do we need S3/SQS/RDS e.t.c. for ECS Task execution role?

Reply:
Right, those are not needed in Task execution role, removed them.

Reply:
Sorry, are they removed? Seems not yet?

Reply:
Further cleaned up.